Healthy/Classes/ReusableViews/CheckboxButton.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+class CheckboxButton: UIButton {
+
+    // Images
+    let checkedImage = UIImage.icCheckboxSelected
+    let uncheckedImage = UIImage.icCheckboxNotSelected
+
+    // Bool property
+    var isChecked: Bool = false {
+        didSet {
+            if isChecked == true {
+                self.setImage(checkedImage, for: UIControl.State.normal)
+            } else {
+                self.setImage(uncheckedImage, for: UIControl.State.normal)
```

Comment:
self.setImage(isChecked ? .icCheckboxSelected : .icCheckboxNotSelected, for: UIControl.State.normal)

Reply:
Done, thanks for reviewing !!

Reply:
I've wrote it as 
```swift
            let image = isChecked ? UIImage.icCheckboxSelected : UIImage.icCheckboxNotSelected
            self.setImage(image, for: UIControl.State.normal)
```